01-intro-to-R.Rmd

```diff
@@ -11,17 +11,19 @@ source("setup.R")
 
 ------------
 
-> ### Learning Objectives
->
-> * Define the following terms as they relate to R: script, function, working directory, assign, object, variable.
-> * Call functions with zero or more named or unnamed arguments.
-> * Solve mathematical operations in R.
-> * Assign values to objects and variables in R.
-> * Describe what vectors are and how they are manipulated in R.
-> * Inspect the content of vectors in R and manipulate their content.
-> * Extract values from vectors in R.
-> * Employ logic (i.e. TRUE, FALSE) to subset data in a vector.
-> * Analyze vectors with missing data.
+> ### Learning Objectives  
+  
+> At the end of this lesson the learner will  be able to:   
+  
+> * Define the following terms as they relate to R: object, assign, call, function, arguments, options.
+> * Create objects and and assign values to them.
+> * Commenting scripts.
```

Comment:
@fmichonneau @achazhardenberg Is there a way we can add a verb to the learning objective on commenting/comment scripts? So basically the objective is for them to use scripts to comment? Is that right? How can we rephrase that so that it's measurable from an assessment perspective?